packages/chain-adapters/src/utxo/UTXOChainAdapter.ts

```diff
@@ -408,7 +413,7 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
       scriptType: accountTypeToScriptType[accountType],
       showDisplay: Boolean(showOnDevice)
     })
-    if (!btcAddress) throw new Error('BitcoinChainAdapter: no btcAddress available from wallet')
+    if (!btcAddress) throw new Error(`UTXOChainAdapter: no ${this.coinName} Address available from wallet`)
```

Comment:
(The naming here is wrong for historical reasons -- all UTXOs use the btcGetAddress function in hdwallet)

```suggestion
    if (!btcAddress) throw new Error('UTXOChainAdapter: no btcAddress available from wallet')
```